modules/security-context-constraints-psa-synchronization.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+:_content-type: CONCEPT
```

Comment:
Maybe Pod Security Admission should have its own new page that would elaborate on global configuration, label synchronization and alerts?

Reply:
@stlaz There are a couple of things to think through, here:

1. Currently, this is all included at the end of `managing-security-context-constraints.adoc`. I'm not opposed to moving new PSA content into a document that exists at the same level as that one, though.
2. Which information that is _required_ for this feature to GA is currently missing? Which information that would be nice to have, but is not critical for GA is missing and could be bug fixed or enhanced later?

Reply:
1. I think this deserves its own place in the docs
2. I don't think there's missing information

Reply:
1. I can try that out today. 
2. These are just:
2a. Label `security.openshift.io/scc.podSecurityLabelSync=false`?
2b.  Not sure. All namespaces that aren't prefixed w/`openshift-`?

Reply:
Actually, 2a should be covered by `modules/security-context-constraints-psa-opting.adoc` at the moment. Do you concur?

Reply:
Ah, true. We are still missing the information that the label sync is disabled on user-created "openshift-"-prefixed namespaces but can be still opted in there, and why.

We are also missing the information that the namespaces that are deployed by the installer cannot be opted in to the sync.

Reply:
Updated.

Reply:
Where is this new documentation, then?

Reply:
@stlaz Actually, don't have this:

> label sync is disabled on user-created "openshift-"-prefixed namespaces but **can be still opted in there, and why.**

I assume the answer to the first part is `security.openshift.io/scc.podSecurityLabelSync=true`, but I'm not sure about "why."

Reply:
The "why" is because those are considered system namespaces and as such, the expectation is that there exists a controller that takes care of them.

The "why" on the installer payload namespaces is because these namespaces are maintained by OpenShift developers.